app/models/match.rb

```diff
@@ -274,23 +275,24 @@ def recalculate
     end
 
     if contest.contest_type == Contest::TYPE_LADDER
-      contester1.extra = contester1.extra + contest.modulus_base/10
-      contester2.extra = contester2.extra + contest.modulus_base/10
+      contester1.extra = contester1.extra + contest.modulus_base / 10
+      contester2.extra = contester2.extra + contest.modulus_base / 10
     end
 
     unless contest.contest_type == Contest::TYPE_BRACKET
-      contester1.score = contester1.score+points1 < 0 ? 0 : contester1.score+points1
-      contester2.score = contester2.score+points2 < 0 ? 0 : contester2.score+points2
+      contester1.score = contester1.score + points1 < 0 ? 0 : contester1.score + points1
+      contester2.score = contester2.score + points2 < 0 ? 0 : contester2.score + points2
       contester1.save!
       contester2.save!
     end
   end
 
-  def hltv_record addr, pwd
-    if (match_time - MATCH_LENGTH*10) > DateTime.now or (match_time + MATCH_LENGTH*10) < DateTime.now
+  def hltv_record(addr, pwd)
+    if (match_time - MATCH_LENGTH * 10) > DateTime.now ||
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.